lib/server.js

```diff
@@ -889,73 +885,18 @@ Server.prototype._run = function _run(req, res, route, chain, cb) {
 
         if (arg) {
             if (arg instanceof Error) {
```

Comment:
`arg` can also be a boolean, or more specifically, a value of `false` which tells restify to stop executing the handler chain.